pkg/oci/containerd.go

```diff
@@ -136,50 +136,49 @@ func verifyStatusResponse(resp *runtimeapi.StatusResponse, configPath string) er
 	return fmt.Errorf("Containerd registry config path is %s but needs to contain path %s for mirror configuration to take effect", cfg.Registry.ConfigPath, configPath)
 }
 
-func (c *Containerd) Subscribe(ctx context.Context) (<-chan ImageEvent, <-chan error) {
-	imgCh := make(chan ImageEvent, 1)
-	errCh := make(chan error, 1)
-
+func (c *Containerd) Subscribe(ctx context.Context) (<-chan ImageEvent, <-chan error, error) {
+	imgCh := make(chan ImageEvent)
+	errCh := make(chan error)
 	client, err := c.Client()
 	if err != nil {
-		errCh <- err
-		close(imgCh)
-		close(errCh)
-		return imgCh, errCh
+		return nil, nil, err
 	}
-
 	envelopeCh, cErrCh := client.EventService().Subscribe(ctx, c.eventFilter)
```

Comment:
`cErrCh` could do with a better name.

Reply:
I agree but its so small it should not matter for now.